src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -586,12 +586,13 @@ public static function identifyUploadedTrack($path)
         }
     }
 
-    public static function identifyAndStoreTrack($tmpid, $title, $artist, $album, $position)
+    public static function identifyAndStoreTrack($tmpid, $title, $artist, $album, $position, $explicit)
```

Comment:
I think there's still some thinking to be done on this option.

I'm currently thinking it has three values, one of which is default (making the parameter optional)
null (default): It is unknown if this track is explicit
true: Yes, it is explicit
false: No, it isn't explicit

confirm_central_upload can work with that, and then future things using identifyAndStoreTrack will hopefully not have ambiguity.